fuzz/Cargo.toml

```
@@ -12,6 +12,8 @@ workspace = true
 cargo-fuzz = true
 
 [dependencies]
+mutatis = { workspace = true }
+bincode = "2.0.1"
```

We should use `postcard` instead of `bincode` here, because we already depend on it in the workspace and don't need to worry about auditing or anything.

```suggestion
postcard = { workspace = true }
```